### ISSUE TITLE
Latest build (3.1.1) fails with TS Error

### ISSUE THREAD

**Issue by farcasmihai91** [AlbertBrand/vue-async-function#9]
Hi,
latest build fails with the following error:

```
Option 'emitDeclarationOnly' cannot be specified without specifying option 'declaration' or option 'composite'.
```

**Comment by AlbertBrand** [comment]
I presume you tried to run `npm run build` from the `/lib` folder? This works for me.

Or did you add it as a dependency in your own project and it now fails? Please show me more than just the error, maybe a repository?

**Comment by AlbertBrand** [comment]
Also [circleCI is green](https://app.circleci.com/pipelines/github/AlbertBrand/vue-async-function/26/workflows/283ce84d-eafc-487f-ad4d-df360d375046/jobs/21). So it might be a problem on your side.

**Comment by farcasmihai91** [comment]
I added it as a dependency in my own project and it now fails. here is the npm install log:

```
 vue-async-function@3.1.1 postinstall C:\projects\project\node_modules\vue-async-function
> npm run build


> vue-async-function@3.1.1 build C:\projects\project\node_modules\vue-async-function
> npm run build:types && npm run build:js


> vue-async-function@3.1.1 build:types C:\projects\project\node_modules\vue-async-function
> tsc --emitDeclarationOnly

error TS5069: Option 'emitDeclarationOnly' cannot be specified without specifying option 'declaration' or option 'composite'.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vue-async-function@3.1.1 build:types: `tsc --emitDeclarationOnly`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vue-async-function@3.1.1 build:types script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\user\AppData\Roaming\npm-cache\_logs\2020-04-02T06_28_27_537Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vue-async-function@3.1.1 build: `npm run build:types && npm run build:js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vue-async-function@3.1.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\user\AppData\Roaming\npm-cache\_logs\2020-04-02T06_28_27_563Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vue-async-function@3.1.1 postinstall: `npm run build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vue-async-function@3.1.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\user\AppData\Roaming\npm-cache\_logs\2020-04-02T06_28_29_121Z-debug.log
command failed: npm install --loglevel error vue-async-function
```

**Comment by AlbertBrand** [comment]
Ah, it seems you have an older version of Typescript in your project, and I incorrectly added a postinstall script that is triggered with your TS version. I'll fix this soon!

**Comment by AlbertBrand** [comment]
Fixed in v3.1.2

**Comment by mihailtd** [comment]
Thank you!
